libs/clients/driving-license/v1/src/lib/drivingLicenseApiV1.module.ts

```diff
@@ -0,0 +1,45 @@
+import { DynamicModule } from '@nestjs/common'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  Configuration,
+  OkuskirteiniApi as OkuskirteiniApiV1,
```

Comment:
Let's subclass the generated API classes and register those instead of using a injection token:

```typescript
import { OkuskirteiniApi } from '../../gen/fetch'
export class OkuskirteiniApiV1 extends OkuskirteiniApi {}
```

Just remember to re-export this class in the root index.ts (and not the original class from `gen/fetch`).